extensions/wikia/PageHeaderExperiments/js/localSearchSuggestions.js

```diff
@@ -0,0 +1,95 @@
+require(['mw', 'jquery', 'wikia.window'], function(mw, $, window) {
+	const experimentId = '7835092457';
+
+	if (window.optimizely.activeExperiments.includes(experimentId) &&
+		window.optimizely.variationMap[parseInt(experimentId)] !== 0) {
+
+		mw.loader.using('jquery.autocomplete').then(function () {
+
+			var $searchInput = $('#searchInputTest');
+			var $searchInputWrapper = $('#searchInputWrapperTest');
+			var searchSuggestionsUrl = $searchInput.data('suggestions-url');
+			var autocompleteReEscape = new RegExp('(\\' + ['/', '.', '*', '+', '?', '|', '(', ')',
+					'[', ']', '{', '}', '\\'].join('|\\') + ')', 'g');
+
+			if ($searchInput === undefined || $searchInputWrapper === undefined) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove this "===" check; it will always be false. Did you mean to use "=="? [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS3403)
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Change this condition so that it does not always evaluate to "false". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2583)